s3_prod_push.sh

```diff
@@ -14,7 +14,6 @@ EXCLUDES="s3_excludes"
 
 openssl aes-256-cbc -K $encrypted_e2db0eb08244_key -iv $encrypted_e2db0eb08244_iv -in secrets.tar.enc -out secrets.tar -d
 tar xvf secrets.tar
-
```

Comment:
The line ending was wrong, so it's removing the line and re-inserting it so that the script will run on Travis.